src/status_im/chat/models/message.cljs

```diff
@@ -94,7 +93,7 @@
   [{:keys [chat-id message-id] :as message} {:keys [now] :as cofx}]
   (handlers-macro/merge-fx cofx
                      (chat-model/upsert-chat {:chat-id chat-id
-                                              ; We activate a chat again on new messages
+                                              ;; We activate a chat again on new messages
```

Comment:
why we need this change in PR this particular PR?

Reply:
Because `;` is only used for after line comments, it's explicitly stated in clojure style guide as well.
My mac is setup to auto indent according to those rules and when, so either I don't use auto-indent (sucks very hard), or I introduce indentation changes when I use it.
Same reason why leaving bad indentation with `handlers-macro` (it's inconsistent even among one file, like this one) is bad idea I think and it should be done right (indentation) on the first time, or in the very next PR touching the file (this one). 
When I leave it as it is, it's just the next unfortunate guy touching the files who will auto-indent (according to clojure style guide we are supposed to adhere to) and introduce "unnecessary changes".

Reply:
Yes that's clear, but you always can create another PR which objective will be style fixes. 
Regarding `handlers-macro` thing: don't worry, i will be that guy and i will file a separate PR with fixed indentations.